sources/web/datalab/polymer/components/table-details/table-details.ts

```diff
@@ -72,41 +72,42 @@ class TableDetailsElement extends Polymer.Element {
       const datasetId = matches[2];
       const tableId = matches[3];
 
-      GapiManager.getTableDetails(projectId, datasetId, tableId)
-        .then((result) => {
-          this._table = JSON.parse(result.body) as gapi.client.bigquery.BigqueryTable;
-        }, (error) => console.log('Failed to get table details: ' + error.body))
+      GapiManager.getBigqueryTableDetails(projectId, datasetId, tableId)
+        .then((response: HttpResponse<gapi.client.bigquery.Table>) => {
+          this._table = JSON.parse(response.body);
```

Comment:
The HttpResponse struct includes a 'result' field that should contain the already-parsed version of 'body'. You should be able to say "this._table = response.result;" here.